### PR TITLE
Merge of Tropical Sea Level POD

### PR DESCRIPTION
**Pull request by jkrasting**
This PR supersedes #107.

**Description**
Include a summary of the change, and link the associated issue (if applicable).  
List any dependencies that are required for this change, including libraries and variables,  
and their metadata (units, frequencies, etc...). Be sure to separate PRs and issues for new PODs and PODs currently under development.

Associated issue # (replace this phrase and parentheses with the issue number)  

**How Has This Been Tested?**
Please describe the tests that you ran to verify your changes in enough detail that  
someone can reproduce them. Include any relevant details for your test configuration  
such as the Python version, package versions, expected POD wallclock time, and the   
operating system(s) you ran your tests on.

**Checklist:**
- [ ] I have reviewed my own code to ensure that if follows the [POD development guidelines](https://mdtf-diagnostics.readthedocs.io/en/latest/sphinx/dev_guidelines.html)
- [ ] My branch is up-to-date with the NOAA-GFDL develop branch, and all merge conflicts are resolved
- [ ] The script are written in Python 3.6 or above (preferred; required if funded by a CPO grant), NCL, or R
- [ ] All of my scripts are in the diagnostics/[POD short name] subdirectory, and include a main_driver script, template html, and settings.jsonc file
- [ ] I have made corresponding changes to the documentation in the POD's doc/ subdirectory
- [ ] If applicable, I've added a .yml file to src/conda, and my environment builds with `conda_env_setup.sh` 
- [ ] I have added any necessary data to input_data/obs_data/[pod short name] and/or input_data/model/[pod short name]
- [ ] My code is portable; it uses [MDTF environment variables](https://mdtf-diagnostics.readthedocs.io/en/latest/sphinx/ref_envvars.html), and does not contain hard-coded file or directory paths
- [ ] I have provided the code to generate digested data files from raw data files
- [ ] Each digested data file generated by the script contains numerical data (no figures), and is 3 GB or less in size
- [ ] The repository contains no extra test scripts or data files

### CODE REVIEW

**Comment by chiaweh2** [comment]
Hi @jkrasting, Thank you for letting me know about the issue! I will start testing and try to find the problem as soon as possible.

**Comment by chiaweh2** [comment]
@jkrasting I was testing the data for a bit and found the synthetic data is only with value over eastern Indian Ocean and part of southern Atlantic Ocean. Do you remember which region you pick to calculate the corresponding value for my POD? I think this might related to the region you picked and if there is value in the synthetic data? I include a figure showing the areacello of your synthetic data below. 
Screen Shot 2021-09-06 at 4 03 01 PM

**Comment by jkrasting** [comment]
Hi @chiaweh2 ... When I plot `areacello`, I see data at all locations.  How did you come to find missing data?

This is `CESM2_omip1_r1i1p1f1_gn.areacello.mon.nc` from the synthetic dataset.

image

**Comment by chiaweh2** [comment]
Hi @jkrasting  I was using the CESM2_omip1_r1i1p1f1_gn.synthetic.zip file you provided above.

**Comment by jkrasting** [comment]
@chiaweh2 - could you please post a snippet so I can reproduce your plot?  I am seeing valid values for all longitudes

**Comment by chiaweh2** [comment]
@jkrasting Thanks for letting me double check. I think when using panoply (netcdf viewing software) it render the lat Lon wrong which cause the weird plot. I check again by using xarray.plot and shows the same result as your plot. I will see if can find the problem. Sorry for deviate a bit from the original issue.

**Comment by chiaweh2** [comment]
@jkrasting I made the changes here on this commit. The problem is the synthetic data is having longitude range of -270-90 which is not the usual OMIP output format I see in original CESM2 which is 0-360. In the new commit I altered the masking array based on 0-360 and mirror it to the corresponding index in -270-90. It seems to solve the problem. Hope this helps! https://github.com/NOAA-GFDL/MDTF-diagnostics/pull/107/commits/953073e77a10cd9296ac7c19dbe97522f3801069

**Comment by jkrasting** [comment]
@chiaweh2 - looks good, thanks!  I tested this out and it seems to work fine.  My only last remaining request would be to "beef" up the text on the `index.html`.   Could you possibly provide a bit more guidance to model developers on how to interpret these plots and what to look for when reading them? 

@wrongkindofdoctor - we are ready to go with merging this in.

File: src/cmip6-cmor-tables/.circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
+version: 2
+
```
Comment (wrongkindofdoctor) [inline src/cmip6-cmor-tables/.circleci/config.yml:2]: @jkrasting the cmip6-cmor-tables directory and files should be deleted from the src directory. They were moved to the data folder in a prior commit, and it looks like this change got lost during the rebase.

File: src/xr_parser.py [hunk 2]
```diff
@@ -146,14 +146,17 @@ def _old_axes_dict(self, var_name=None):
         coords_list = list(axes_obj.coords)
```
Comment (wrongkindofdoctor) [inline src/xr_parser.py:146]: Remove the changes to xr_parser.py. I believe the fix ended up being addressed elsewhere in PR #185

**Review by wrongkindofdoctor — CHANGES_REQUESTED** [review]

**Comment by wrongkindofdoctor** [comment]
@jkrasting I submitted a [PR](https://github.com/jkrasting/MDTF-diagnostics/pull/1) to your branch with the requested changes

**Comment by jkrasting** [comment]
Thanks @wrongkindofdoctor - I just merged your PR onto my branch.

**Review by wrongkindofdoctor — APPROVED** [review]

**Pull request merged**